src/tests/specs/iteration.spec.ts

```diff
@@ -26,7 +26,18 @@ describe('Iteration test', () => {
     await planner.iteration.clickCreateIteration();
     expect(await planner.sidePanel.getIterationDate()).toContain('new Iteration [Active]'+month+' 1, '+year+' - '+month+' '+ lastDayOfMonth, +year+'\n');
   });
-
+  
+  it('should create a new child iteration', async () => {
+    let newIteration = 'new Iteration';
+    let parentIteration = '/' + process.env.SPACE_NAME + '/Iteration_2';
+    let iteration2 = 'Iteration_2';
+    await planner.sidePanel.createNewIteration();
+    await planner.iteration.addNewChildIteration(newIteration, parentIteration);
+    await planner.iteration.clickCreateIteration();
+    await planner.sidePanel.clickExpander(iteration2);
+    expect(await planner.sidePanel.getIterationList()).toContain(newIteration);
+  }
```

Comment:
@divyanshiGupta test is failing with an exception...Please add }); with this test

Reply:
@Raunak1203 Sorry for the typo. Updated the commit.